xoinvader/ingame.py

```diff
@@ -93,6 +94,12 @@ def __init__(self, owner):
         self._objects = []
         self._screen = self._owner.screen
 
+        self.bg = Background(Settings.path.level1bg)
+        self.bg.start(True)
+        self.bg.speed = 10
```

Comment:
Yes. I don't know  though.
Also I completely forgot about `@property` decorator. Should I wrap public fields in it?